### DISCUSSION TITLE
Welcome to edl Discussions!

### DISCUSSION THREAD

**Discussion by bkerler** [bkerler/edl#385] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cyber-bro47** [comment]
Hello. Please fix your links LiveDVD. 
[ https://github.com/bkerler/edl/?tab=readme-ov-file#use-livedvd-everything-ready-to-go-based-on-ubuntu](https://github.com/bkerler/edl/?tab=readme-ov-file#use-livedvd-everything-ready-to-go-based-on-ubuntu)
